src/main/java/org/alliancegenome/curation_api/jobs/executors/Gff3Executor.java

```diff
@@ -61,30 +70,27 @@
 				}
 			}
 			gffData.subList(0, gffHeaderData.size()).clear();
-
 			
 			BulkFMSLoad fmsLoad = (BulkFMSLoad) bulkLoadFile.getBulkLoad();
 			BackendBulkDataProvider dataProvider = BackendBulkDataProvider.valueOf(fmsLoad.getFmsDataSubType());
 
-			Map<String, List<Long>> idsAdded = new HashMap<String, List<Long>>();
-			idsAdded.put("Transcript", new ArrayList<Long>());
-			idsAdded.put("Exon", new ArrayList<Long>());
-			idsAdded.put("CodingSequence", new ArrayList<Long>());
-			idsAdded.put("TranscriptGenomicLocationAssociation", new ArrayList<Long>());
-			idsAdded.put("ExonGenomicLocationAssociation", new ArrayList<Long>());
-			idsAdded.put("CodingSequenceGenomicLocationAssociation", new ArrayList<Long>());
+			List<ImmutablePair<Gff3DTO, Map<String,String>>> preProcessedGffData = preProcessGffData(gffData, dataProvider);
```

Comment:
🚫 **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.WhitespaceAfterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>',' is not followed by whitespace.
<!-- __reviewdog__:ChAxZjAyMWE0NTllOTYwZDU5EgpjaGVja3N0eWxl -->